package.json

```diff
@@ -78,6 +78,7 @@
     "randomstring": "^1.1.0",
     "rc": "~0.5.0",
     "request": "^2.51.0",
+    "request-promise": "^1.0.2",
```

Comment:
Ok cook. Yeah, that should work. Can you point me to example? I'm pretty new to promises fyi